setup.cfg

```diff
@@ -1,3 +1,15 @@
+[isort]
```

Comment:
@pfmoore I think you missed that there's a new Travis CI job that uses the additional tox's "isort" command that has been added... 

I guess the churn in `.travis.yml` was the reason. (my bad)